connector_cli/connectorpy.py

```diff
@@ -244,7 +248,7 @@ def collapse_connector(
                 should_warn = True
         if should_warn:
             warning_text = (
-                "WARNING! There is no use for template parameter(s) "
+                "WARNING! There is no use for template parameter(s)"
```

Comment:
The extra space should be here otherwise there is no space, it's linked to the f-string underneath

Reply:
I know